src/components/SideNavigation/SideNavigation.scss

```diff
@@ -167,4 +169,9 @@ body .childLi {
             }
         }
     }
+
+    .bodyCollapsed {
+        z-index: -10;
```

Comment:
@Farhan-emgage I'd like to revisit this application of z-index.

Reply:
@weegrom instead of z index applied display property